packages/cli/src/commands/experimental/templates/realtime.ts.template

```diff
@@ -21,13 +21,16 @@ import subscriptions from 'src/subscriptions/**/*.{js,ts}'
  * Redwood Realtime
  *  - uses a publish/subscribe model to broadcast data to clients.
  *  - uses a store to persist Live Query and Subscription data.
+ *  - and enable defer and stream directives to improve latency
+ *    for clients by sending data the most important data as soon as it's ready.
  *
  * Redwood Realtime supports in-memory and Redis stores:
  * - In-memory stores are useful for development and testing.
  * - Redis stores are useful for production.
  *
  */
 export const realtime: RedwoodRealtimeOptions = {
+  enableDeferStream: true,
```

Comment:
We have this `enableDeferStream` here and in a commented out form a few lines down. Did we mean to have two?